providers/implementations/rands/drbg.c

```diff
@@ -513,7 +522,7 @@ static int ossl_prov_drbg_reseed_unlocked(PROV_DRBG *drbg,
         }
         if (ent_len > drbg->max_entropylen) {
             ERR_raise(ERR_LIB_RAND, RAND_R_ENTROPY_INPUT_TOO_LONG);
-            drbg->state = EVP_RAND_STATE_ERROR;
+            drbg->state = EVP_RAND_STATE_ERROR;int oss
```

Comment:
This seems like you left a trailing variable declaration in the code.

Reply:
Fixed, but I cannot sign the document until Monday.